google/ggrc-core
# GGRC-6318 Revision for IssuetrackerIssue 'create' action is absent

GGRC-6318 Revision for IssuetrackerIssue 'create' action is absent for bulk ticket generation

# Issue description

When user press 'Generate' button on Audit page we generate new tickets for each Assessment in Audit. If any Assessment here doesn't have related IssueTrackerIssue, such object will be created automatically. But revision for this created IssueTrackerIssue is missed (with event).
Expected result: We should always have 'create' revision (and event) for IssueTrackerIssue object.

# Steps to test the changes
In UA:

- Create new Audit with disabled IssueTracker
- Create one or more Assessment corresponding to this Audit
- Enable IssueTracker in Audit
- Press Generate button
- Check DB for corresponding revisions

Inside dev container:

<pre>
<code>
1. cd test
2. python -m unittest -v integration.ggrc.integrations.test_bulk_issue_generate.TestBulkIssuesChildGenerate.test_issuetracker_issue_revision_creation
</code>
</pre>

# Solution description

Added function that creates event and revisions for each newly created  IssuetrackerIssue.

# Sanity checklist


# PR Review checklist
